File: app/src/main/java/com/widyatama/nurseassistant/features/main/MainPresenter.java
package com.widyatama.nurseassistant.features.main;

import com.widyatama.core.base.BasePresenter;
import com.widyatama.core.util.SchedulerProviderUtil;

import io.reactivex.disposables.CompositeDisposable;

public class MainPresenter<V extends MainViewContracts> extends BasePresenter<V> implements MainPresenterContracts<V> {

    public MainPresenter(SchedulerProviderUtil schedulerProvider, CompositeDisposable compositeDisposable){
        super(schedulerProvider, compositeDisposable);
    }
}

File: app/src/main/java/com/widyatama/nurseassistant/features/main/MainPresenter.kt
package com.widyatama.nurseassistant.features.main

import com.widyatama.core.base.BasePresenter
import com.widyatama.core.util.SchedulerProviderUtil
import io.reactivex.disposables.CompositeDisposable

class MainPresenter<V: MainViewContracts>
constructor(schedulerProvider:
SchedulerProviderUtil, compositeDisposable: CompositeDisposable) : BasePresenter<V>(
        schedulerProvider, compositeDisposable), MainPresenterContracts<V> {
}